src/tests/class_tests/openms/source/AASequence_test.cpp

```diff
@@ -232,8 +232,17 @@ START_SECTION((double getAverageWeight(Residue::ResidueType type = Residue::Full
 END_SECTION
 
 START_SECTION((double getMonoWeight(Residue::ResidueType type = Residue::Full, Int charge=0) const))
-  TOLERANCE_ABSOLUTE(1e-6)
-  TOLERANCE_RELATIVE(1.0 + 1e-6)
+  TOLERANCE_ABSOLUTE(1e-5)
+  TOLERANCE_RELATIVE(1.0 + 1e-5)
```

Comment:
we compare the result to the output of an online tool (see the discussion in the issue #1310).
Somehow they seem to calculate masses slightly differently but don't know the exact reason (e.g. maybe by subtracting electrons from neutrons instead of calculating with protons as we do)